### PR TITLE
Cypress test have been added

### PR DESCRIPTION
**Pull request by murat-yesimbietov**

### CODE REVIEW

File: cypress/integration/page.spec.js [hunk 1]
```diff
@@ -1,7 +1,114 @@
...
+Cypress.Commands.add('getButton', (text) => {
+  cy.contains('button', text);
+});
+
+Cypress.Commands.add('checkGoodsList', (items) => {
+  cy.get('li').each((li, index) => {
+    cy.wrap(li)
+      .should('contain', items[index]);
+  });
+});
```
Comment (YuriiHoliuk) [inline cypress/integration/page.spec.js:31]: краще просто об'єкт з методами робити, глянь як в Яріка

File: cypress/integration/page.spec.js [hunk 2]
```diff
@@ -1,7 +1,114 @@
...
+  it(`should show the goods in the initial order after clicking on 'Reset'`, () => {
+    cy.getButton(buttons.sortByLength).click();
+
+    cy.getButton(buttons.sortAlph).click();
+
+    cy.getButton(buttons.reset).click();
+
+    cy.checkGoodsList(goodsFromServer);
+  });
+});
```
Comment (YuriiHoliuk) [inline cypress/integration/page.spec.js:114]: add an empty line in the end

**Review by YuriiHoliuk — APPROVED** [review]

File: cypress/integration/page.spec.js (hunk 1 above)
Comment (murat-yesimbietov) [reply cypress/integration/page.spec.js]: done

File: cypress/integration/page.spec.js (hunk 2 above)
Comment (murat-yesimbietov) [reply cypress/integration/page.spec.js]: done

File: README.md [hunk 3]
```diff
@@ -4,12 +4,12 @@
...
 
 ## Tasks
-1. At first show a button `Start` on the page
-1. After clicking the button show a `GoodsList` (ul > li) with the given goods and hide the button
-1. Add `Reverse` button to show the goods in reversed order
-1. Add `Sort alphabetically` button to show the goods in alphabetical order
-1. Add `Reset` button to show the goods in the initial order
-1. Add `Sort by length` button to show the goods ordered by name length
+1. At first show a button with text `Start` on the page
+1. After clicking the button shows a `GoodsList` (ul > li) with the given goods and hide the `Start` button
```
Comment (YevhS) [inline README.md:8]: something is wrong with grammar here) 'after clicking the button, the list is shown ...' maybe? or if it's an instruction  - 'after clicking the button show ...'

**Review by YevhS — APPROVED** [review]

**Pull request merged**